addtocalendar.module

```diff
@@ -26,9 +26,14 @@ function addtocalendar_help($route_name, RouteMatchInterface $route_match) {
  */
 function addtocalendar_field_formatter_third_party_settings_form($plugin, $field_definition, $view_mode, $form, $form_state) {
   $element = [];
-  if (count(array_intersect(['datetime', 'daterange'], $plugin->getPluginDefinition()['field_types']))) {
-    $settings = $plugin->getThirdPartySettings('addtocalendar');
-    $element = _addtocalendar_build_form($settings, $field_definition);
+  // dsm(get_class_methods($plugin));
+  // dsm($plugin->getPluginId());
+  if($plugin->getPluginId() !== 'add_to_calendar'){
+    //make sure to avoid confliction in case add_to_calendar field formatter is used
```

Comment:
Coding Standard 
// Make sure to avoid confliction in case add_to_calendar field formatter is used.